jdbc/src/main/scala/zio/sql/jdbc.scala

```diff
@@ -193,7 +192,14 @@ trait Jdbc extends zio.sql.Sql {
           tryDecode[java.util.UUID](
             java.util.UUID.fromString(column.fold(resultSet.getString(_), resultSet.getString(_)))
           )
-        case TZonedDateTime      => ???
+        case TZonedDateTime      =>
+          tryDecode[java.time.ZonedDateTime](
+            java.time.ZonedDateTime
+              .ofInstant(
+                column.fold(resultSet.getTimestamp(_), resultSet.getTimestamp(_)).toInstant,
+                ZoneId.of(ZoneOffset.UTC.getId)
```

Comment:
I'm not sure how we'd like to handle this, having it default to utc seems problematic, but I'm not sure how we should go from java.sql.Timestamp -> java.time.ZonedDateTime.

Reply:
looking at the javadoc of getTimestamp, I'm pretty sure that it's converting to UTC, so I believe this is correct. It'd probably be a good idea to add a round trip test to make sure (e.g. insert a timestamp from some other time zone, grab it back out, then compare to ensure that time is the same (accounting for the difference in time zones)

Reply:
Yeah that's a good idea, I'll add that in!